### PR TITLE
[Docs] fixed order

### PR DESCRIPTION
**Pull request by OskarStark**
Use the following order for yaml service registrations:
 - class
 - arguments
 - calls
 - tags
 - public

### CODE REVIEW

**Comment by core23** [comment]
This is symfony best practice?

**Comment by OskarStark** [comment]
At least we generate this order and Symfony does the same order, not sure it is considered best practice

File: docs/cookbook/recipe_persisting_filters.rst [hunk 1]
```diff
@@ -73,16 +73,12 @@ Per Admin :
...
-                        name: sonata.admin
-                        manager_type: orm
-                        filter_persister: filter_persister_service_id
                 arguments:
                     - ~
                     - AppBundle\Entity\User
                     - ~
-
+                tags:
+                    - { name: sonata.admin manager_type: orm filter_persister: filter_persister_service_id }
```
Comment (greg0ire) [inline docs/cookbook/recipe_persisting_filters.rst]: Please rever to multiline style, this is probably too long for docs and will make a horizontal scroll appear. Also, you forgot a few commas in there :P

File: docs/reference/advanced_configuration.rst [hunk 2]
```diff
@@ -68,7 +68,13 @@ You have 2 ways of defining the dependencies inside ``services.xml``:
...
                 - ~
             tags:
-                - { name: sonata.admin, manager_type: orm, group: "Project", label: "Project", label_translator_strategy: "sonata.admin.label.strategy.native",  route_builder: "sonata.admin.route.path_info" }
+                -
+                    name: sonata.admin
+                    manager_type: orm
+                    group: 'Project'
+                    label: 'Project'
+                    label_translator_strategy: 'sonata.admin.label.strategy.native'
+                    route_builder: 'sonata.admin.route.path_info'
```
Comment (greg0ire) [inline docs/reference/advanced_configuration.rst]: 👍

**Comment by greg0ire** [comment]
Thanks @OskarStark !

**Pull request merged**